tests/console/validate_vnc_avail.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that after remote installation, the VNC connection
+# between client and host, is available without the use of vnc password.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use mmapi;
+use y2lan_restart_common "close_xterm";
+
+# Returns the first leased IP, in /var/lib/dhcp/db/dhcpd.leases file
+sub get_lease_ipaddr {
+    my $lease_ip;
+    my $dhcp_leases_file = script_output("cat /var/lib/dhcp/db/dhcpd.leases\n");
+    for my $line (split /\n/, $dhcp_leases_file) {
+        if ($line =~ /^lease\s+\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/) {
+            (undef, $lease_ip) = split(/\s/, $line);
+            return $lease_ip;
+        }
+    }
+}
+
+sub start_vncviewer {
+    my $args = shift;
```

Comment:
by convention, `my ($args)`.

Reply:
OK. When was that convention set? With a grep in the repo I see "my $args" being more frequently used that "my ($args)".

Reply:
you're right, that convention is for objects, only in the case `my ($self) = shift;` I believe that for expansion in case the object accept other params, but in this case you just pass a reference which is scalar. Thanks for checking it.